internal/cmd/kafka/command_broker_describe.go

```diff
@@ -53,7 +38,7 @@ func (c *brokerCommand) newDescribeCommand() *cobra.Command {
 }
 
 func (c *brokerCommand) describe(cmd *cobra.Command, args []string) error {
-	brokerId, all, err := checkAllOrBrokerIdSpecified(cmd, args)
+	brokerId, all, err := broker.CheckAllOrBrokerIdSpecified(cmd, args)
```

Comment:
Since this is in the `broker` package, probably don't need to write "broker" in the function name
```suggestion
	brokerId, all, err := broker.CheckAllOrIdSpecified(cmd, args)
```

Reply:
In the future, I want to get rid of the `--all` flag. For example,

```confluent kafka broker configuration list --all```

should probably be written as:

```confluent kafka cluster configuration list```

Reply:
(Not sure how you're using `--all` in the Confluent Local commands, but maybe we should consider removing it pre-emptively)